mitodl/bootcamp-ecommerce
# Add Amount to Pay and Award Cost custom fields

#### What are the relevant tickets?
Related to #236

#### What's this PR do?
Add Amount to Pay and Award Cost custom fields

#### How should this be manually tested?
Set up an Award Cost custom field for your program, trigger a webhook  and make sure that is the amount the user has to pay.

<img width="1214" alt="screen shot 2018-12-05 at 4 14 20 pm" src="https://user-images.githubusercontent.com/7574259/49544477-eed60e00-f8a8-11e8-913c-780e7d793f49.png">

Repeat the same with Amount to Pay custom field that is set for the application/submission.
(Go to the program's dashboard, click on Applications tab, click the application id, edit custom fields)
